### ISSUE TITLE
Doesn't work with node v14.0.0

### ISSUE THREAD

**Issue by lstorey** [jdalrymple/gitbeaker#2331]
**Description**

All requests return an empty object `{}` with node version 14.0.0, both with the CLI and node library.

**Steps to reproduce**

```
nvm install 14.0.0
nvm use 14.0.0
gitbeaker projects all --gl-token=<your-gl-token>
```

**Expected behaviour**

Returns a list of projects

**Actual behaviour**

Returns an empty object: `{}`

**Possible fixes**

Consider specifying a minimum node version in the README and/or the package.json

**Comment by jdalrymple** [comment]
Which version of gitbeaker are you testing with?

**Comment by lstorey** [comment]
35.1.0

**Comment by jdalrymple** [comment]
Ill give it a look!

**Comment by jdalrymple** [comment]
Oh and youre referring only to gitbeaker/cli correct? or also gitbeaker/node

**Comment by lstorey** [comment]
Both are producing the same behavior.

**Comment by lstorey** [comment]
BTW I'm not blocked on this at all, I just upgraded my node version to the
latest (17.4.0) and got everything working. Just hoping to save someone
else the hassle of tracking this down if they have the same issue in the
future.

**Comment by jdalrymple** [comment]
100% i should probably use the correct engine information in the pkg.json. Thanks for the report!

**Comment by jdalrymple** [comment]
Weird behaviour started at 14.2.0!

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`35.3.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/35.3.0) :rocket:
